client/src/components/AddReviewModal.jsx

```diff
@@ -1,21 +1,49 @@
 import React from 'react';
 import Stars from './Stars.jsx';
+import $ from 'jquery';
+import { IMGBB_KEY, EMAIL_VALIDATION_API_KEY } from '../../../config';
```

Comment:
I like the change that you made, but in the feature we'll need to change `config.example.js` to reflect the new format. For now, it's something Sung and I can edit on the main branch.